.github/stale.yml

```diff
@@ -44,10 +44,12 @@ pulls:
 
 issues:
   markComment: >
-    This issue has been automatically marked as stale because it has not had
-    recent activity. It will be closed if no further activity occurs. Thank you
+    Hello! This issue has been automatically marked as stale because it has not had
```

Comment:
Nice touch with the thank you msg 👍 

wording here is a bit confusing since we're asking questions that can have different answers:
* Has your question been answered? **No**
* Do you still need more information? **Yes**

which leads to the closure description being unclear. An attempt at consolidating the questions into one:

`Hello! This issue has been automatically marked as stale because it has not had any recent activity. Have your concerns been addressed? If no, leave a comment and someone from the team can take a look. Otherwise, the issue will be automatically closed if no further activity occurs. Thank you for your contributions.`

Reply:
I agree. This is better wording.